SU-SWS/saa_victoria_subtheme
# Adapt 367  top banner no image & with image - both red backgrounds

# READY FOR REVIEW

# Summary
- Added styles for top banner red background & no-image fallback version

# Review By (Date)
- 6/15

# Urgency
- low

# Steps to Test

1. pull down branch
1. view a page with a banner with text and with an image, should look nice with bright red background on left 50%, according to figma
2. then delete the image or view a banner with text fields but no image. styles should all be the same except bright red background goes 100% full width of the banner.

# Affected Projects or Products
just saa subtheme


# Associated Issues and/or People
- ADAPT-367 no image
- ADAPT-356 image and text
